app/client/src/features/events/EventLive.tsx

```diff
@@ -125,8 +124,17 @@ export function EventLive({ eventLiveQueryRef, validateInviteQueryRef }: EventLi
         <EventContext.Provider value={{ eventId: node.id, isModerator: Boolean(node.isViewerModerator) }}>
             <Grid component={motion.div} key='townhall-live' container className={classes.root} onScroll={handleScroll}>
                 {!isMdUp && <div ref={topRef} />}
-                <Grid item md={8} className={classes.video}>
-                    <EventVideo fragmentRef={node} />
+                <Grid
+                    container item
+                    md={8}
+                    direction='column'
+                    wrap='nowrap'
+                >
+                    <Grid item className={classes.video}>
+                        <EventVideo fragmentRef={node} />
+                    </Grid>
+                    <EventDetailsCard fragmentRef={data} />
+                    <SpeakerList fragmentRef={data} />
```

Comment:
If you update the eventLiveQueryRef you should be able to pass it in here to replace data.